system/ee/ExpressionEngine/Controller/Design/AbstractDesign.php

```diff
@@ -322,7 +322,7 @@ protected function exportTemplates()
         // Create the Zip Archive
         $zipfilename = tempnam(sys_get_temp_dir(), '');
         $zip = new ZipArchive();
-        if ($zip->open($zipfilename, ZipArchive::CREATE) !== true) {
+        if ($zip->open($zipfilename, ZipArchive::OVERWRITE) !== true) {
```

Comment:
Would it be better to use `ZipArchive::CREATE | ZipArchive::OVERWRITE` in this instance?

Reply:
Why would be :: OVERWRITE not sufficient?

Reply:
I was under the impression that `::OVERWRITE` only works when there is a file to overwrite, and that you needed to use `::CREATE` if you wanted to create a file. 

See: https://stackoverflow.com/questions/49164547/ziparchive-i-need-to-create-new-file-and-overwrite-existing

Reply:
We are creating file with `tempnam` here, so it's always there

Reply:
Okay thats what I was missing. Sounds good!